frontend/src/app/modules/autocompleter/create-autocompleter/create-autocompleter.component.ts

```diff
@@ -104,8 +106,11 @@ export class CreateAutocompleterComponent extends UntilDestroyedMixin implements
     this.onAfterViewInit.emit(this);
     if (this.opInviteUserModalService) {
       this.opInviteUserModalService.close
-        .pipe(this.untilDestroyed())
-        .subscribe((user: HalResource) => {
+        .pipe(
+          this.untilDestroyed(),
+          filter(user => !!user)
```

Comment:
This is an unrelated fix I noticed. When cancelling the modal, the assignee field was still saved and a notification was shown.